goreleaser_v1.yaml

```diff
@@ -3,9 +3,12 @@ builds:
       - CGO_ENABLED=0
     flags: []
     goos:
-      - darwin
       - linux
       - windows
+      - darwin
+      - freebsd
+      - openbsd
+      - solaris
```

Comment:
Go supports only solaris/amd64 builds. You'll need to add other combinations to the ignore list.